3ds Max/Max2Babylon/BabylonSkipFlatten.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.Windows.Forms;
+using Autodesk.Max;
+using ActionItem = Autodesk.Max.Plugins.ActionItem;
+
+namespace Max2Babylon
+{
+    class BabylonInvertSkipFlatten:ActionItem
+    {
+
+        public override bool ExecuteAction()
+        {
+            IINode sel = Loader.Core.GetSelNode(0);
+            if (sel == null) return true;
+
+            bool skip = sel.IsMarkedAsNotCollapsable();
+            sel.SetUserPropBool("babylonjs_SkipFlatten", !skip);
+            return true;
+        }
+
+        public void Close()
+        {
+            return;
+        }
+
+        public override int Id_
+        {
+            get { return 1; }
+        }
+
+        public override string ButtonText
+        {
+            get { return "Babylon Invert Skip Flatten Status"; }
+        }
+
+        public override string MenuText
+        {
+            get
+            {
+                IINode sel = Loader.Core.GetSelNode(0);
+                if (sel == null)
+                {
+                    return "&Babylon Skip Flatten";
```

Comment:
Can we change the menu text to something more intuitive like

"Scene Flattening - Enabled"
and
"Scene Flattening - Disabled"

Reply:
ok I agree